src/Orchard.Web/Modules/Orchard.Email/Services/SmtpMessageChannel.cs

```diff
@@ -116,7 +106,11 @@ public class SmtpMessageChannel : Component, ISmtpChannel, IDisposable {
                     mailMessage.ReplyTo.AddRange(ParseRecipients(emailMessage.ReplyTo));
                 }
 
-                _smtpClientField.Value.Send(mailMessage);
+                using (var smtpClient = CreateSmtpClient()) {
```

Comment:
Might be worth having an overload which takes multiple emails to reuse the client, otherwise bulk-sending them will create as many connections.